versions.tsp

```diff
@@ -0,0 +1,9 @@
+import "@typespec/versioning";
+
+@Versioning.versioned(OpenAIFlavors)
+namespace ModelDefinitions;
+
+enum OpenAIFlavors {
+    OpenAI,
+    Azure,
```

Comment:
All models in Azure should be super-sets of OpenAI. And if they are not, we could accommodate that with marking them as `@removed(OpenAIFlavors.Azure)`.